<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ColumnsManDlgClass</class>
 <widget class="QDialog" name="ColumnsManDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ColumnsManDlg</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTextEdit" name="textEdit" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="colmd_lw_cols" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="colmd_pb_show" >
         <property name="text" >
          <string>&amp;Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colmd_pb_hide" >
         <property name="text" >
          <string>&amp;Hide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colmd_pb_move_up" >
         <property name="text" >
          <string>Move &amp;Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colmd_pb_move_down" >
         <property name="text" >
          <string>Move Do&amp;wn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colmd_pb_default" >
         <property name="text" >
          <string>&amp;Default</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>The selected column should be</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="colmd_le_col_width" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>pixel width.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsManDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel" >
     <x>110</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsManDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>347</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
